backend/routes/domains.js

```diff
@@ -0,0 +1,68 @@
+const { Resolver } = require('dns')
+
+const { authSellerAndShop, authRole } = require('./_auth')
+
+const { getLogger } = require('../utils/logger')
+
+const log = getLogger('routes.domains')
+
+const resolver = new Resolver()
+resolver.setServers(['8.8.8.8', '8.8.4.4'])
+
+module.exports = function (app) {
+  app.post(
+    '/domains/verify-dns',
+    authSellerAndShop,
+    authRole('admin'),
+    async (req, res) => {
+      const { domain, hostname } = req.body
+
+      log.debug('Trying to verify DNS records of domain', domain, hostname)
+
+      try {
+        const cnameData = await new Promise((resolve, reject) => {
+          resolver.resolveCname(domain, (err, data) =>
+            err ? reject(err) : resolve(data)
+          )
+        })
+
+        const cnameValid = cnameData.includes('ipfs-prod.ogn.app')
```

Comment:
We need to stop using this hardcoded value.  How about using `network.ipfs` from the network table, since that's what's being used in the DNS records assignments now?

Reply:
Thanks @mikeshultz. Will make these changes